front/app/components/admin/PostManager/components/LazyPostPreview/Idea/IdeaContent.test.tsx

```diff
@@ -39,7 +39,7 @@ jest.mock('components/T', () => 'T');
 jest.mock('services/ideas');
 jest.mock('resources/GetResourceFiles', () => 'GetResourceFiles');
 jest.mock('resources/GetIdea', () => 'GetIdea');
-jest.mock('resources/GetIdeaImages', () => 'GetIdeaImages');
+jest.mock('hooks/useIdeaImages', () => 'useIdeaImages');
```

Comment:
I'm not sure if this mock makes sense. For the resource, you're replacing a component by a string, so I guess that's equivalent to rendering a string in the JSX- but hooks are not rendered in JSX, so here you would be trying to 'call' a string